### PR TITLE
Fix address generator issue #530

### PR DESCRIPTION
**Pull request by iketheadore**

### CODE REVIEW

File: cmd/address_gen/address_gen.go [hunk 1]
```diff
@@ -24,6 +25,8 @@ func main() {
 	genCount := flag.Int("n", 1, "Number of addresses to generate")
 	hideSecKey := flag.Bool("s", false, "Hide the secret key from the output")
 	isBitcoin := flag.Bool("b", false, "Print address as a bitcoin address")
+	hexSeed := flag.Bool("x", false, "Use hex string as seed")
+	verbose := flag.Bool("v", false, "Show verbose info of generated addresses")
```
Comment (gz-c) [inline cmd/address_gen/address_gen.go]: Change `verbose` to `j,json`?

File: cmd/address_gen/address_gen.go [hunk 2]
```diff
@@ -24,6 +25,8 @@ func main() {
 	genCount := flag.Int("n", 1, "Number of addresses to generate")
 	hideSecKey := flag.Bool("s", false, "Hide the secret key from the output")
 	isBitcoin := flag.Bool("b", false, "Print address as a bitcoin address")
+	hexSeed := flag.Bool("x", false, "Use hex string as seed")
```
Comment (gz-c) [inline cmd/address_gen/address_gen.go]: What is hex string seed for

File: cmd/address_gen/address_gen.go (hunk 1 above)
Comment (iketheadore) [reply cmd/address_gen/address_gen.go]: It's for showing details of generated address info, including pubkey, seckey, seed, etc. So I think verbose is ok.

File: cmd/address_gen/address_gen.go (hunk 2 above)
Comment (iketheadore) [reply cmd/address_gen/address_gen.go]: the generator use random hex string as default seed before, we just keep it here.

**Pull request merged**